### DISCUSSION TITLE
Optimal Usage of svelte french toast: Is it Feasible to Import Once and Reuse Everywhere?

### DISCUSSION THREAD

**Discussion by ezaz-ahmed** [kbrgl/svelte-french-toast#50] [General]
## Discussion: Optimal Usage of svelte french toast: Is it Feasible to Import Once and Reuse Everywhere?

Hey everyone,

I've been working on a project that involves using the `<Toaster />` plugin for toast notifications. Currently, I've been including the `<Toaster />` component in each individual component where I need toast functionality. While this works, I'm wondering if there's a more efficient way to achieve this without repeating the same code across multiple components.

**Exploration:**

I'm eager to discuss and explore alternative strategies to optimize the usage of the `<Toaster />` plugin. Specifically, I'd like to know if there's a way to import the toast functionality once, perhaps at a higher level in the app, and then access it seamlessly across all components without the need to include the `<Toaster />` component in each one.

**Question:**

Do I need to manually include `<Toaster />` in every component where I want to display toast notifications?

Looking forward to hearing from you all!

**Comment by kbrgl** [comment]
No, the recommended usage is mounting the `<Toaster />` component at the root of your app. You do not need to include it with every use.
